src/TTrigger.cpp

```diff
@@ -461,11 +463,13 @@ END : {
         pC->reset();
     }
     if (mIsMultiline) {
-        updateMultistates(regexNumber, captureList, posList);
+        NameGroupMatches* ng = &nameGroups;
+        updateMultistates(regexNumber, captureList, posList, ng);
```

Comment:
This would work just as well:

```suggestion
        updateMultistates(regexNumber, captureList, posList, &nameGroups);
```